apps/ltn/src/edit/map_edits.rs

```diff
@@ -0,0 +1,76 @@
+use map_model::{EditRoad, MapEdits, RoadID};
```

Comment:
Moved from the one_way code. This just handles `MapEdits`. The LTN tool doesn't fully use this format (which the traffic simulator does), because modal filters aren't defined there.